highlighting/definition.py

```diff
@@ -25,6 +25,7 @@
     "def",
     "return",
     "print",
+    "play",
```

Comment:
You don't need to edit these files anymore, since they are related to Ace, and after #4924 is merged, we wouldn't need to keep mainting this files.